dnsapi/dns_cx.sh

```diff
@@ -82,7 +82,7 @@ existing_records() {
     return 1
   fi
   count=0
-  seg=$(printf "%s\n" "$response" | _egrep_o "{[^\{]*host\":\"$_sub_domain\"[^\}]*\}")
+  seg=$(printf "%s\n" "$response" | _egrep_o "{[^{]*host\":\"$_sub_domain\"[^}]*}")
```

Comment:
Hum.... It doesn't work on my ArchLinux. It cannot match the response text.
I'll make a test under freebsd later.